src/adapters/github.less

```diff
@@ -23,6 +23,7 @@
 
   .ui-resizable-e {
     right: -4px;
+    width: 0px;
```

Comment:
Would this make it extremely difficult to resize? I mean previously width is 8px, there's a lot of space for users to hover and drag. Not sure if I miss anything.